e2e/test/scenarios/filters/reproductions/34794-filter-popover-navigation-error.cy.spec.js

```diff
@@ -0,0 +1,31 @@
+import {
+  filter,
+  getNotebookStep,
+  openOrdersTable,
+  popover,
+  restore,
+} from "e2e/support/helpers";
+
+describe("issue 34794", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+  });
+
+  it("should not crash when navigating to filter popover's custom expression section", () => {
```

Comment:
```suggestion
  it("should not crash when navigating to filter popover's custom expression section (metabase#34794)", () => {
```

Reply:
Actually I'm not sure it makes sense to add issue number again in the `it` section, but I saw it in many other places